code401challenges/src/test/java/utilities/MultiBracketValidation.java

```diff
@@ -3,13 +3,41 @@
 import stacksandqueues.Stack;
 
 public class MultiBracketValidation {
-    Stack<Character> stack = new Stack<>();
 
     public static boolean multiBracketValidation(String input){
+        Stack<Character> validationStack = new Stack<>();
+        char validation = ' ';
 
+//  iterates through each character in my string
+        for(char charChar : input.toCharArray()){
+//  If we get a opening bracket push it into the stack
+            if( charChar == '(' || charChar == '[' || charChar == '{'){
+                validationStack.push(charChar);
+            }
+//  If we get a closing bracket pop
+            if (charChar == ')') {
```

Comment:
You might have been able to combine these nested `if` statements into one conditional:
`if (charChar == ')' && validationStack.pop() != '(')`
This isn't necessary but I think combining nested ifs into one makes it a little easier to read.